samples/azure/custom-audit-transport/Core_9/AuditViaASQ/AuditViaASQFeatureStartup.cs

```diff
@@ -0,0 +1,46 @@
+﻿using NServiceBus.Features;
+using NServiceBus.Transport;
+
+namespace AuditViaASQ;
+
+class AuditViaASQFeatureStartup() :
+    FeatureStartupTask
+{
+    public static IMessageDispatcher? AsqDispatcher;
+
+    private TransportInfrastructure? transportInfrastructure;
+
+    protected override async Task OnStart(IMessageSession session, CancellationToken cancellationToken)
+    {
+        var hostName = $"{nameof(AuditViaASQFeature)} host";
+
+        var hostSettings = new HostSettings(
+            hostName,
+            hostName,
+            new StartupDiagnosticEntries(),
+            OnCriticalError,
+            true,
+            null);
+
+        var transport = new AzureStorageQueueTransport("UseDevelopmentStorage=true");
+        transport.DelayedDelivery.DelayedDeliveryPoisonQueue = "audit-via-asq-delayed-poison";
```

Comment:
Is there an option to disable delayed delivery to not have to set this config value?

Reply:
It seems to be [required for sendonly endpoints](https://github.com/Particular/NServiceBus.AzureStorageQueues/blob/master/src/Transport/AzureStorageQueueTransport.cs#L311-L316).